src/pages/dar_application/ResearcherInfo_new.js

```diff
@@ -196,12 +194,15 @@ export default function ResearcherInfo(props) {
             Internal Collaborators will not be required to submit an independent DAR to collaborate 
             on this project.`
           ),
-          button({
-            type: 'button', // default button element type inside a form is "submit".
-            className: 'button button-white',
-            style: { marginTop: 25 },
-            onClick: () => {}
-          }, ['Add Collaborator'])
+          h(CollaboratorList, {
+            formFieldChange,
+            collaborators: internalCollaborators,
+            collaboratorKey: 'internalCollaborators',
+            collaboratorLabel: 'Internal Collaborator',
+            showApproval: false,
+            disabled: !isEmpty(darCode),
```

Comment:
outside of this PR scope, but I think how we determine "is this DAR a draft" is no longer if the darCode is empty... i am not sure how it is though. i think this is ok to merge for now, but this disable condition might change in the future.